core/benches/kura.rs

```diff
@@ -42,15 +42,18 @@ async fn measure_block_size_for_n_validators(n_validators: u32) {
         iroha_core::kura::Kura::new(iroha_config::kura::Mode::Strict, dir.path(), false).unwrap();
     let _thread_handle = iroha_core::kura::Kura::start(kura.clone());
 
-    let block = PendingBlock::new(vec![tx], Vec::new())
-        .chain_first()
-        .validate(
-            &TransactionValidator::new(transaction_limits),
-            &Arc::new(WorldStateView::new(World::new(), kura)),
-        );
-    let mut block = block
-        .sign(KeyPair::generate().expect("Failed to generate KeyPair"))
-        .unwrap();
+    let mut block = PendingBlock::new(
+        vec![tx],
+        Vec::new(),
+        1,
+        None,
+        0,
+        iroha_core::sumeragi::network_topology::Topology::new(Vec::new()),
+        KeyPair::generate().expect("Failed to generate KeyPair"),
+        &TransactionValidator::new(transaction_limits),
+        &Arc::new(WorldStateView::new(World::new(), kura)),
```

Comment:
In python we'd use named parameters (`**kwargs`). Here, the next best thing is the struct literal syntax. 

```rust
PendingBlockStub {
   transactions: Vec<VersionedAcceptedTransaction>,
   event_recommendations: Vec<Event>,
   height: u64,
   previous_block_hash: Option<HashOf<VersionedCommittedBlock>>,
   view_change_index: u64,
   committed_with_topology: Topology,
   key_pair: KeyPair,
   transaction_validator: &TransactionValidator,
   wsv: &WorldStateView,
}
```
Where you get the benefit of not worrying about filling in all the details thanks to the ability to only define the struct fields that matter and using `..PendingBlockStub::default()` later. This allows you not to have to specify things like  height (mostly it's `1` unless otherwise specified) and `view_change_index`. 

Finally, you just `impl From<PendingBlockStub> for PendingBlock` and postfix the call to the struct literal with `.into()`.

Reply:
After looking at the `new` method, it might be a good idea to just add 
```rust
impl PendingBlockStub { 
  fn build() -> PendingBlock {
  ```